hazelcast-enterprise/Dockerfile

```diff
@@ -50,18 +73,18 @@ ENV CLASSPATH ""
 ENV JAVA_OPTS ""
 
 # Start Hazelcast server
-CMD ["bash", "-c", "set -euo pipefail \
-      && if [[ \"x${CLASSPATH}\" != \"x\" ]]; then export CLASSPATH=\"${CLASSPATH_DEFAULT}:${CLASSPATH}\"; else export CLASSPATH=\"${CLASSPATH_DEFAULT}\"; fi \
-      && if [[ \"x${JAVA_OPTS}\" != \"x\" ]]; then export JAVA_OPTS=\"${JAVA_OPTS_DEFAULT} ${JAVA_OPTS}\"; else export JAVA_OPTS=\"${JAVA_OPTS_DEFAULT}\"; fi \
-      && if [[ \"x${MIN_HEAP_SIZE}\" != \"x\" ]]; then export JAVA_OPTS=\"${JAVA_OPTS} -Xms${MIN_HEAP_SIZE}\"; fi \
-      && if [[ \"x${MAX_HEAP_SIZE}\" != \"x\" ]]; then export JAVA_OPTS=\"${JAVA_OPTS} -Xms${MAX_HEAP_SIZE}\"; fi \
-      && if [[ \"x${HZ_LICENSE_KEY}\" != \"x\" ]]; then export JAVA_OPTS=\"${JAVA_OPTS} -Dhazelcast.enterprise.license.key=${HZ_LICENSE_KEY}\"; fi \
-      && if [[ \"x${MANCENTER_URL}\" != \"x\" ]]; then export JAVA_OPTS=\"${JAVA_OPTS} -Dhazelcast.mancenter.enabled=true -Dhazelcast.mancenter.url=${MANCENTER_URL}\"; fi \
-      && echo \"########################################\" \
-      && echo \"# JAVA_OPTS=${JAVA_OPTS}\" \
-      && echo \"# CLASSPATH=${CLASSPATH}\" \
-      && echo \"# starting now....\" \
-      && echo \"########################################\" \
-      && set -x \
-      && exec java -server ${JAVA_OPTS} com.hazelcast.core.server.StartServer \
+CMD ["bash", "-c", "set -euo pipefail && \
```

Comment:
I suppose this is just a matter of personal preference, I actually find the ampersands at BOL just equally readable :-) but no objections to change.

Reply:
@googlielmo , I always used the formatting with the "&&" and the end, but after your comments, checked how they use is on the official Docker page. And actually, they use them at the beginning, so I reverted the formatting change. Thanks for the comment.